src/test/java/com/rackspace/ceres/app/web/QueryControllerTest.java

```diff
@@ -24,22 +28,28 @@
 import org.springframework.boot.test.autoconfigure.web.reactive.AutoConfigureWebFlux;
 import org.springframework.boot.test.autoconfigure.web.reactive.AutoConfigureWebTestClient;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.TestConfiguration;
 import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Bean;
 import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.web.reactive.server.WebTestClient;
 import reactor.core.publisher.Flux;
 import reactor.test.StepVerifier;
 
 @ActiveProfiles(profiles = {"test", "query"})
-@SpringBootTest(classes = {QueryController.class, AppProperties.class, RestWebExceptionHandler.class,
-    DownsampleProperties.class})
+@SpringBootTest(classes = {QueryController.class, AppProperties.class,
+    RestWebExceptionHandler.class,
+    DownsampleProperties.class, MeterRegistryConfig.class})
```

Comment:
Instead of using `MeterRegistryConfig` here you can simply add `SimpleMeterRegistry` here only.